modules/assim.sequential/R/Multi_Site_Constructors.R

```diff
@@ -6,9 +6,6 @@
 ##' @param var.names vector names of state variable names.
 ##' @param X a matrix of state variables. In this matrix rows represent ensembles, while columns show the variables for different sites.
 ##' @param localization.FUN This is the function that performs the localization of the Pf matrix and it returns a localized matrix with the same dimensions.
-##' @param t not used
-##' @param blocked.dis passed to `localization.FUN`
-##' @param ... passed to `localization.FUN`
```

Comment:
All parameters need Roxygen entries

Reply:
...But if you have better descriptions for the parameters, please do improve my wording!

Reply:
Fixed.